packages/synapse-react-client/src/components/ChangePassword/ChangePassword.integration.test.tsx

```diff
@@ -495,11 +501,6 @@ describe('ChangePassword tests', () => {
         userId,
         twoFaToken,
       ),
-      // Update the mock server so the next request will fail with a meaningful error
-      getBadRequestChangePasswordHandler(
-        getEndpoint(BackendDestinationEnum.REPO_ENDPOINT),
-        errorMessage,
-      ),
```

Comment:
I modified the behavior of the other handler in this test, so I had to relocate this handler to be loaded at a later point in the test.